fe/src/project/Seller.Portal/areas/Clients/components/ClientForm/ClientForm.js

```diff
@@ -173,6 +181,21 @@ function ClientForm(props) {
                                 )}
                             </FormControl>
                         </div>
+                        <div className="field">
```

Comment:
@SebaKas21 please use the regular TextField here. I don't want to hardcode regions like "europe" here.

Reply:
@dawiddworak88 ReactPhoneInput should be deleted?